### PR TITLE
Improve travis

### PR DESCRIPTION
**Pull request by orangejulius**
Do all the things listed in pelias/pelias#258

Connects pelias/pelias#258

### CODE REVIEW

File: test/importPipeline.js [hunk 1]
```diff
@@ -15,11 +15,6 @@ tape('functional test of importing two small OA files', function(t) {
   var expected = JSON.parse(fs.readFileSync(expectedPath));
 
   var endStream = event_stream.writeArray(function(err, results) {
-    // calling toJSON on a Document writes the phrase, so we have to emulate that here
```
Comment (dianashk) [inline test/importPipeline.js]: why was this removed?

Comment (orangejulius) [inline test/importPipeline.js]: Hmm, I don't remember how this ended up in this PR. The code isn't needed any more since pelias-model now handles setting phrase. I'll move it to a separate PR.

**Comment by dianashk** [comment]
:100:
